packages/components/psammead-caption/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-caption",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
I'm surprised that the previous PRs didn't catch these. That's quite worrying! I wonder if the mismatched package/package-lock versions published on npm will cause `npm ci` to error. 
I suppose it'll be fixed next time we publish these packages to npm.